data/Vishnu-Biju.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Vishnu Biju",
+  "bio": "I'm a skilled full stack developer with experience in JavaScript,MongoDb and expertise in frameworks like React, Node.js, and Three.js.",
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/Vishnu-Biju/",
+      "icon": "FaGithub"
+    },
+     {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/InVishnubiju",
+      "icon": "FaTwitter"
+    },
+      {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/vishnu-biju1/",
+      "icon": "FaLinkedin"
+    },
```

the comma needs to be removed:
```suggestion
    }
```